src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -25,24 +31,92 @@
 
     public static class MapLoader {
 
+
+        /**
+         * Create a NumericsSample from the given column of the
+         * in the TreeMap
+         */
+        public static NumericSample buildSample(TreeMap<String, List> dataSet, String colName){
```

Comment:
I don't exactly get what this method is doing in this specific class.

Reply:
This function is meant to extract as a  ```NumericSample``` the column in the given ```dataSet``` named ```colName```. In fact I have change the name of the function to ```buildNumericSample``` in the other branch because I have a similar one for ```CategoricalSample```. Also I am thinking of changing ```TreeMap<String, List>``` to ```TreeMap<String, List<String>>``` because ```CSVParser``` actually returns strings.

Reply:
I think the implantation of the `csvParser` should change to hashMap. check the comment below.